AOJ/06/06_B.py

```diff
@@ -0,0 +1,13 @@
+n = int(input())
+allcard = list()   
+patern = ['S', 'H', 'C', 'D']
+for egara2 in patern:
+    for suuji2 in range(1 , 14):
+        allcard.append(egara2 + ' ' + str(suuji2))
```

Comment:
```suggestion
allcard = [f'{egara2} {suuji2}' for egara2 in patern for suuji2 in range(1, 14)]
```
- 内包型とf-stringで書くと短く書くことができます。
- 特に、`egara2 + ' ' + str(suuji2) >>> f'{egara2} {suuji2}'` で、`str`で囲む手間を省くことができます。